docs/content/preview/architecture/transactions/transaction-priorities.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Transaction priorities
+headerTitle: Transaction priorities
+linkTitle: Transaction priorities
+description: Details about Transaction priorities in YSQL
+menu:
+  preview:
+    identifier: architecture-transaction-priorities
+    parent: architecture-acid-transactions
+    weight: 1153
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../transaction-priorities/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+
+</ul>
+
+When using the [Fail-on-Conflict](../concurrency-control/#fail-on-conflict) concurrency control policy, transactions are assigned priorities that help decide which transactions should be aborted on conflicts.
```

Comment:
"which transactions should be aborted on conflicts."
->
"which transactions should be aborted in case of conflict."